source/receiver.cpp

```diff
@@ -22,12 +23,24 @@ bool Receiver::add_inlink(std::shared_ptr<ILink> link) {
         LOG_WARN("Link destination device is incorrect (expected current device)");
         return false;
     }
-    m_router->add_inlink(link);
-    return true;
+    return m_router->add_outlink(link);

```

Comment:
I think it ment to be inlink here:
```suggestion
    return m_router->add_inlink(link);

```

Reply:
Of course, my bad